bpf/lib/nat.h

```diff
@@ -570,8 +571,7 @@ snat_v4_nat(struct __ctx_buff *ctx, const struct ipv4_nat_target *target,
 }
 
 static __always_inline __maybe_unused int
-snat_v4_rev_nat(struct __ctx_buff *ctx, const struct ipv4_nat_target *target,
-		bool from_endpoint)
+snat_v4_rev_nat(struct __ctx_buff *ctx, const struct ipv4_nat_target *target)
```

Comment:
We have two versions of `snat_v4_rev_nat()`, but you forgot to update the arguments for the other one (in the other branch of `#if defined(ENABLE_IPV4) && defined(ENABLE_NODEPORT)`). Please update it, too